docs/getting-started/provision-infrastructure.md

```diff
@@ -0,0 +1,218 @@
+with the `MySQLServer` CRD it installs into your cluster.
+
+> Note: provisioning an Azure Database for MySQL requires the presence of a
+> [Resource Group] in your Azure account. We go ahead and provision a new
+> `ResourceGroup` here in case you do not already have a suitable one in your
+> account.
+
+```yaml
+apiVersion: azure.crossplane.io/v1alpha3
+kind: ResourceGroup
+metadata:
+  name: sqlservermysql-rg
+spec:
+  location: West US 2
+  reclaimPolicy: Delete
+  providerRef:
+    name: azure-provider
+---
+apiVersion: database.azure.crossplane.io/v1beta1
+kind: MySQLServer
+metadata:
+  name: sqlservermysql
+spec:
+  forProvider:
+    administratorLogin: myadmin
+    resourceGroupNameRef:
+      name: sqlservermysql-rg
+    location: West US 2
+    sslEnforcement: Disabled
+    version: "5.7"
+    sku:
+      tier: GeneralPurpose
+      capacity: 2
+      family: Gen5
+    storageProfile:
+      storageMB: 20480
+  writeConnectionSecretToRef:
+    namespace: crossplane-system
+    name: sqlservermysql-conn
+  providerRef:
+    name: azure-provider
+  reclaimPolicy: Delete
+```
+
+Creating the above instance will cause Crossplane to provision a MySQL database
+instance on Azure. You can view the progress with the following command:
+```
+kubectl get mysqlservers.database.azure.crossplane.io sqlservermysql
+```
+
+When provisioning is complete, you should see `READY: True` in the output. You
+can then delete the `MySQLServer`:
+```
+kubectl delete mysqlservers.database.azure.crossplane.io sqlservermysql
+```
+
+</div>
+<div class="tab-pane fade" id="alibaba-tab-1" markdown="1">
+
+The Alibaba provider supports provisioning an [AsparaDB for RDS] instance with
+the `RDSInstance` CRD it installs into your cluster.
+```yaml
+apiVersion: database.alibaba.crossplane.io/v1alpha1
+kind: RDSInstance
+metadata:
+  name: rdsmysql
+spec:
+  forProvider:
+    engine: mysql
+    engineVersion: "5.7"
+    dbInstanceClass: rds.mysql.s1.small
+    dbInstanceStorageInGB: 20
+    securityIPList: "0.0.0.0/0"
+    masterUsername: "test123"
+  writeConnectionSecretToRef:
+    namespace: crossplane-system
+    name: alibaba-rdsmysql-conn
+  providerRef:
+    name: alibaba-provider
+  reclaimPolicy: Delete
+```
+
+Creating the above instance will cause Crossplane to provision an RDS instance
+on Alibaba. You can view the progress with the following command:
+```
+kubectl get rdsinstances.database.alibaba.crossplane.io rdsmysql
+```
+
+When provisioning is complete, you should see `READY: True` in the output. You
+can then delete the `RDSInstance`:
+```
+kubectl delete rdsinstances.database.alibaba.crossplane.io rdsmysql
+```
+</div>
+</div>
+
+## Next Steps
+
+Now that you have seen how to provision individual infrastructure resources,
+let's take a look at how we can compose infrastructure resources together and
+publish them as a single unit to be consumed in the [next section].
+
+<!-- Named Links -->
+
+[installed a provider]: install.md
```

Comment:
Should this link to the quickstart version instead? ( docs/getting-started/install-configure.md )